include/dirac_quda.h

```diff
@@ -93,10 +95,11 @@ namespace quda {
       halo_precision(QUDA_INVALID_PRECISION),
       need_bidirectional(false),
 #ifdef QUDA_MMA_AVAILABLE
-      use_mma(true),
+      setup_use_mma(true),
 #else
-      use_mma(false),
+      setup_use_mma(false),
 #endif
+      dslash_use_mma(true),
```

Comment:
Does this intentionally disagree with the default from the command line (`false`)?

Reply:
https://github.com/lattice/quda/pull/1355/commits/ef5c9921a50a31bb86a0ed08739cb17ea6a85cca